event_tracker.gemspec

```
@@ -1,38 +1,39 @@
-# coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'event_tracker/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "event_tracker"
+  spec.name          = 'event_tracker'
   spec.version       = EventTracker::VERSION
-  spec.authors       = ["tom"]
-  spec.email         = ["tom@returnly.com"]
+  spec.authors       = ['tom']
+  spec.email         = ['tom@returnly.com']
```

Tom has not been working at Returnly for a couple of years now
```suggestion
  spec.authors       = ['Richard Millan']
  spec.email         = ['richard@returnly.com']
```

---

Do we have any 'department' email list for those purposes?